packages/node/src/lib/subscribers/subscriber.interface.ts

```diff
@@ -34,7 +34,7 @@ export interface ISubscribers {
    */
   unsetCredentials(subscriberId: string, providerId: string);
   updateOnlineStatus(subscriberId: string, online: boolean);
-  getPreference(subscriberId: string);
+  getPreference(subscriberId: string, includeInactiveChannels: boolean);
```

Comment:
```suggestion
  getPreference(subscriberId: string, { includeInactiveChannels: boolean });
```

Reply:
Passing an object is more scalable and allows future addition of more parameters.